weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -440,7 +440,6 @@ function buildCallsTableColumns(
     align: 'center',
     sortable: false,
     resizable: false,
-    disableColumnMenu: true,
```

Comment:
Doesn't seem to be a good reason to not allow user to pin/unpin or hide the user column from the column menu.